openssl.go

```diff
@@ -103,6 +103,7 @@ var (
 	providerNameFips    = C.CString("fips")
 	providerNameDefault = C.CString("default")
 	propFIPS            = C.CString("fips=yes")
+	propNoFIPS          = C.CString("-fips")
```

Comment:
I'm not able to find any docs on `-fips`, and the lack of symmetry vs. `fips=yes` is a bit strange. Where does this come from?

Reply:
Not sure why it didn't show up in my initial searches, but found the meaning here: https://docs.openssl.org/3.0/man7/property/

Reply:
Arguably the name could be `ignoreGlobalFIPS` to match the docs, but I don't know if that's meaningfully different.